src/pages/partners/affiliate-ib/_calculated-formula.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+    FormulaBottomWrapper,
+    FormulaContainer,
+    FormulaContainerMobileOneLine,
+    FormulaGreen,
+    FormulaHighlight,
+    FormulaResultGreen,
+    FormulaResult,
+    FormulaTopWrapper,
+    FormulaValueMobile,
+    FormulaValueMobileOneLine,
+    FormulaValueResultSwapSynthetic,
+    FormulaValueSwapSynthetic,
+    PointerContainer,
+    PointerContainerMobile,
+    PointerDot,
+    PointerStick,
+    PointerText,
+    PointerTextMobile,
+} from './_style'
+import { Show } from 'components/containers'
+
+const CalculatedFormula = ({ data }) => {
+    return (
+        <>
+            <Show.Desktop>
+                {data.list.map((value, index) => (
+                    <FormulaContainer key={index}>
+                        <FormulaHighlight>
+                            {value.formula.map((list, indexData) => (
+                                <>
+                                    <FormulaValueSwapSynthetic key={indexData}>
+                                        {list.item}
+                                        <PointerContainer>
+                                            <PointerDot />
+                                            <PointerStick />
+                                            <PointerText>{list.description}</PointerText>
+                                        </PointerContainer>
+                                    </FormulaValueSwapSynthetic>
+                                    {list.next_operator ? (
+                                        <FormulaValueSwapSynthetic>
+                                            <FormulaGreen>{list.next_operator}</FormulaGreen>
+                                        </FormulaValueSwapSynthetic>
+                                    ) : (
+                                        ''
+                                    )}
+                                </>
+                            ))}
+                            <FormulaResult>
+                                <FormulaValueResultSwapSynthetic>
+                                    <FormulaResultGreen>=</FormulaResultGreen>
+                                    {value.result.total}
+                                    {value.result.description ? value.result.description : ''}
+                                </FormulaValueResultSwapSynthetic>
+                            </FormulaResult>
+                        </FormulaHighlight>
+                    </FormulaContainer>
+                ))}
+            </Show.Desktop>
+            <Show.Mobile>
+                {data.list.map((value, index) => (
+                    <>
+                        {value.totalItem > 2 ? (
```

Comment:
please make this into a named variable

Reply:
Amended in the latest commit.. Added `const totalNumberofOperation = 2` variable..